pkg/account/deployer/client.go

```diff
@@ -67,6 +67,20 @@ func (d *accountManagementClient) getGlobalPolicies(ctx context.Context) (map[st
 	return result, nil
 }
 
+func (d *accountManagementClient) getAllGroups(ctx context.Context) (map[string]remoteId, error) {
+	groups, resp, err := d.client.GroupManagementAPI.GetGroups(ctx, d.accountInfo.AccountUUID).Execute()
+	defer closeResponseBody(resp)
+	if err = d.handleClientResponseError(resp, err, "unable get all groups for account "+d.accountInfo.AccountUUID); err != nil {
+		return nil, err
+	}
+	result := make(map[string]remoteId)
+	for _, glP := range groups.GetItems() {
+		result[glP.Name] = glP.GetUuid()
```

Comment:
Any reason to use the getter on one site but the property on the other? (glP.Name vs getUuid). Property access should be enough, right?

Reply:
Uuid is of type `*string`